pkg/deploy/generator/resources_rp.go

```diff
@@ -1645,7 +1645,6 @@ func (g *generator) database(databaseName string, addDependsOn bool) []*arm.Reso
 							},
 							Kind: mgmtdocumentdb.PartitionKindHash,
 						},
-						DefaultTTL: to.Int32Ptr(7 * 86400), // 7 days
```

Comment:
The OpenShiftVersions database has already been deployed in production with a setting of 7 days expiry therefore, due to the default incremental behaviour of ARM templates, we need to specify `-1` here to [switch to infinite TTL](https://docs.microsoft.com/en-us/azure/cosmos-db/sql/time-to-live#time-to-live-for-containers-and-items). Otherwise they will be left at 7 days afaik.